west.yml

```diff
@@ -61,11 +61,6 @@ manifest:
       path: modules/hal/atmel
       groups:
         - hal
-    - name: hal_cypress
```

Comment:
Hi @npal-cy ,

You need add hal_infineon at same time you drop hal_cypress.
Remember that you need add your PR number at revision.

revision: pull/2/head

Reply:
correct. in description i added note about dependency on: https://github.com/zephyrproject-rtos/hal_infineon/pull/2.
So when we merge hal_infineon/pull/2, i will add one more commit with update hal_infineon in west.yaml

Reply:
May I suggest you create another PR only with this dependency?
Maybe we can get that faster if that is the only necessary dependency right now.

Reply:
This is the PR that should get https://github.com/zephyrproject-rtos/hal_infineon/pull/2 into the tree @nandojve. Later we will then continue with https://github.com/zephyrproject-rtos/zephyr/pull/44211, but first let's get this merged.